backend/pokemon/helpers.py

```diff
@@ -29,6 +39,12 @@ def create_pokemon(pokemon_data):
     )
 
 
+def pokemon_in_api(poke_name):
+    url = POKE_API_URL + poke_name
+    response = requests.head(url)
+    return bool(response)
```

Comment:
Did you check if this is enough? Try with a non-existing pokemon.

Reply:
I tried. If I type a pokemon that is not in the API, it says "ERROR: Choose only existing Pokemon."